packaging/install/cluster-operator/042-Crd-strimzipodset.yaml

```diff
@@ -68,9 +68,9 @@ spec:
                   matchLabels:
                     x-kubernetes-preserve-unknown-fields: true
                     type: object
-                description: Selector is a label query which matches all the pods
-                  managed by this `StrimziPodSet`. Only `matchLabels` is supported.
-                  If `matchExpressions` is set, it will be ignored.
+                description: "Selector is a label query which matches all the pods\
+                  \ managed by this `StrimziPodSet`. Only `matchLabels` is supported.\
+                  \ If `matchExpressions` is set, it will be ignored."
```

Comment:
I don't think this formatting should be here. It will probably cause some issues.

Reply:
I have seen similar kind of formatting in yaml under `documentation/configuring/modules/install/cluster-operator/042-Crd-strimzipodset.yaml`

I can update the code base at:
```
public class CrdGenerator {
    public static final YAMLMapper YAML_MAPPER = new YAMLMapper()
```
To generate description as single, as found in the Helm chart: `helm-charts/helm3/strimzi-kafka-operator/crds/042-Crd-strimzipodset.yaml`

Is that acceptable @scholzj ?

Reply:
Yeah, basically, ideally, the YAMLs should not change just because we change Jackson version.

Reply:
@scholzj  Looks like this pull request for handling null in YAML has introduced change in behaviour between version 2.11.x and 2.12.x of Jackson

https://github.com/FasterXML/jackson-dataformats-text/issues/253

I applying an explicit formatting in the Crd Generator, and pushing the changes. Hopefully, it will keep CRDs unchanged as Jackson versions get further updated.